src/Ryujinx.Ava/UI/ViewModels/DownloadableContentManagerViewModel.cs

```diff
@@ -120,6 +127,9 @@ public DownloadableContentManagerViewModel(VirtualFileSystem virtualFileSystem,
 
         private void LoadDownloadableContents()
         {
+            // Try to load DLCs from PFS first
```

Comment:
```suggestion
            // NOTE: Try to load downloadable contents from PFS0 first.
```

(or `Partition File System`)